backend/src/jwt/encode.js

```diff
@@ -4,7 +4,7 @@ import CONFIG from './../config'
 // Generate an Access Token for the given User ID
 export default function encode(user) {
   const token = jwt.sign(user, CONFIG.JWT_SECRET, {
-    expiresIn: 24 * 60 * 60 * 1000, // one day
+    expiresIn: '1d',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-23T06:25:36Z" title="Friday, August 23rd 2019, 8:25:36 am +02:00">Aug 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,7 +4,7 @@ import CONFIG from './../config'
 // Generate an Access Token for the given User ID
 export default function encode(user) {
   const token = jwt.sign(user, CONFIG.JWT_SECRET, {
-    expiresIn: 24 * 60 * 60 * 1000, // one day
+    expiresIn: '1d',
```


I think this is more human readable and doesn't need a comment to explain it, but how is this fixing the expiration date? both are valid, correct?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-23T07:53:24Z" title="Friday, August 23rd 2019, 9:53:24 am +02:00">Aug 23, 2019</time>_

---


The was a misconception about the meaning of this number. It's **seconds** not milliseconds. So the previous expiration was after 1000 days.